tests/test_kirsch.py

```diff
@@ -0,0 +1,9 @@
+import centrosome.kirsch
+import skimage.io
+import skimage.data
+import scipy.misc
+import numpy
+
+
+def test_kirsch():
+    assert numpy.array_equal(centrosome.kirsch.kirsch(skimage.data.camera()), scipy.misc.imread("tests/resources/kirsch.tiff"))
```

Comment:
camera() has type uint8 and it's overflowing in scipy.ndimage.convolve:

    >>> from skimage.data import camera
    >>> camera().dtype
    dtype('uint8')

If you do centrosome.kirsch.kirsch(skimage.data.camera().astype(np.float32)), you get the following:

![image](https://cloud.githubusercontent.com/assets/660469/11405671/3fcc08fe-9377-11e5-846a-e44e3274d8db.png)